docker/_include.sh

```diff
@@ -10,7 +10,6 @@ XDG_CACHE_HOME=${XDG_CACHE_HOME:-$HOME/.cache}
 PROJECT_NAME="commcarehq"
 
 function web_runner() {
-    sudo \
-        env DOCKER_DATA_HOME=$DOCKER_DATA_HOME XDG_CACHE_HOME=$XDG_CACHE_HOME\
-        docker-compose -f $DOCKER_DIR/compose/docker-compose-hq.yml -p $PROJECT_NAME $@
+    env DOCKER_DATA_HOME=$DOCKER_DATA_HOME XDG_CACHE_HOME=$XDG_CACHE_HOME\
+        sudo docker-compose -f $DOCKER_DIR/compose/docker-compose-hq.yml -p $PROJECT_NAME $@
```

Comment:
Ah that makes sense. I did add myself to the docker group, but kept `sudo` in there since I figured it was required for travis. didn't realize those variables wouldn't get passed in